examples/md-flexible/tests/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+if (NOT AUTOPAS_BUILD_TESTS)
+    message(STATUS "AUTOPAS_BUILD_TESTS is disabled. Not building MDFlex tests!")
+    return()
+endif ()
```

Comment:
@FG-TUM: I've had to disable the md-flex tests in case of AUTOPAS_BUILD_TESTS=OFF, as it depends on things defined in the autopas tests.
Is it intended that way or are the dependencies unwanted?

Reply:
We never thought about having the example tests without the actual tests but I guess it would make sense that this is possible.

Reply:
=> #569